### PR TITLE
[iOS] Set correct app version string

### LINKED ISSUE #401: [iOS] Set correct App Version string
**Idea Description**

- Currently, app version shows constant string. Please apply correct app version for that.

### PR DESCRIPTION
**Pull request by NakaokaRei**
## Issue
- close #401 

## Overview (Required)
- I implemented to show correct app version.

## Links
- 

## Screenshot
Before | After
:--: | :--:
image | image

### CODE REVIEW

File: app-ios/Modules/Sources/Model/Extension/Bundle.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//
+//  Bundle.swift
+//
+//
+//  Created by rei.nakaoka on 2023/08/16.
+//
+
```
Comment (ry-itto) [inline app-ios/Modules/Sources/Model/Extension/Bundle.swift:7]: Please remove this file header 🙏🏼 
I'll add this SwiftLint rule later 🙇🏼

**Review by ry-itto — APPROVED** [review]
Thank you! Almost looks good!
Please fix only one point 🙏🏼

File: app-ios/Modules/Sources/Model/Extension/Bundle.swift (hunk 1 above)
Comment (NakaokaRei) [reply app-ios/Modules/Sources/Model/Extension/Bundle.swift]: I fixed at https://github.com/DroidKaigi/conference-app-2023/pull/534/commits/ced0096b78d06155e5efe9be2fb66fa9045cfd1d !

**Comment by ry-itto** [comment]
@NakaokaRei 
Thank you for fixing! I'll merge this 🙏🏼

**Pull request merged**